src/shared/error-handlers.js

```diff
@@ -0,0 +1,8 @@
+export const showErrorNotification = error => {
+  const board = document.querySelector('.info-message')
+  const message = document.querySelector('.show-info-message')
+  message.innerText = error.response.data.error.message;
+  board.style.display = 'block';
+  const block = () => board.style.display = 'none';
+  setTimeout(block, 3000)
+}
```

Comment:
all constants/variables should be in one place

Reply:
fixed